libs/client/repository/src/lib/dashboard/tags/tags.service.ts

```diff
@@ -27,6 +27,49 @@ export class TagsService {
         );
     }
 
+    public fetchTagsSortedByParent(kind: TagKind): Observable<TagsModel[]> {
+        return this.network.fetchTagsSortedByParent(kind).pipe(
+            tap(value => {
+                var map = new Map<String, TagsTree>();
```

Comment:
two nits:
1) use `let` instead of `var`: `var` has [weird scoping behavior](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var). I'd call this a general rule of thumb
2) call the variable `tagRoots` or something descriptive